src/styles/Button.module.scss

```diff
@@ -1,42 +1,49 @@
 @import './Constants.scss';
 
+@mixin disableDefaultButtonStyles {
+  font-family: inherit;
+  background: none;
+  border: none;
+  padding: 0;
+}
+
+$hover-transition-time: 0.2s;
+$click-transition-time: 0.01s;
+$button-height: 3.25rem;
+
 .background {
   background-color: black;
 }
 
 .button {
-  background: none;
-	border: none;
-	padding: 0;
-	outline: inherit;
-  display: inline-block;
-  margin: 1rem 1rem;
-  box-sizing: border-box;
+  @include disableDefaultButtonStyles;
   cursor: pointer;
-  width: fit-content;
+  background: none;
+  margin: 1rem;
   font-weight: 700;
-  
+
   &.filled {
     background-color: $primary-color;
     transition: $hover-transition-time;
-    height: 3.25rem;
-    padding: 0.1rem;
+    border: 0.1rem solid $primary-color;
     color: $neutral-color-95;
```

Comment:
Ten `height: 3.25rem` póki co usunąłem, żeby wielkość naszego guzika nie była stała, a żeby była zależna od rozmiaru tekstu, który się w nim znajduje (jest to jeszcze do poprawy, bo wyżej w kodzie sam wrzuciłem zmienną o wysokosci guzika)